omod/src/main/resources/config.xml

```diff
@@ -126,34 +131,176 @@
+			install those two modules to enable pix error handling functionality.
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pdq.errorHandler.implementation</property>
+		<defaultValue></defaultValue>
+		<description>
+			Specify a Spring bean name of PDQ error handler service. It requires the xds-sender module and outgoing message
+			exception module developed by SolDevelo. Set value to "outgoingmessageexceptions.PdqErrorHandlingService" and
+			install those two modules to enable pix error handling functionality.
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pdqEndpoint</property>
+		<defaultValue>http://localhost:5001/pdq</defaultValue>
+		<description>
+			Specifies endpoint for Hl7 PDQ Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pdqPort</property>
+		<defaultValue>3600</defaultValue>
+		<description>
+			Specifies port for Hl7 PDQ Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pixEndpoint</property>
+		<defaultValue>http://localhost:5001/pix</defaultValue>
+		<description>
+			Specifies the endpoint for the Hl7 PIX Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.pixPort</property>
+		<defaultValue>3700</defaultValue>
+		<description>
+			Specifies port for Hl7 PIX Message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.Hl7implementation</property>
+		<defaultValue>registrationcore.mpiHl7v2HttpSender</defaultValue>
+		<description>
+			Specifies which bean should be used to send the Hl7 Message. Current options are:
+			"registrationcore.mpiHl7v2HttpSender", "registrationcore.mpiHl7v2MllpSender".
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.sendingApplication</property>
+		<defaultValue>OpenMRS</defaultValue>
+		<description>
+			Specifies the sending application in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+	
+	<globalProperty>
+		<property>${project.parent.artifactId}.birthdateEstimationStartMonth</property>
+		<defaultValue>0</defaultValue>
+		<description>
+			The month from which the birth date is estimated, with January having a value of 0
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.sendingFacility</property>
+		<defaultValue>Demo</defaultValue>
+		<description>
+			Specifies sending Facility in the MSH header section of the HL7 message
+		</description>
+	</globalProperty>
+
+	<globalProperty>
+		<property>${project.parent.artifactId}.mpi.receivingApplication</property>
+		<defaultValue>Demo MPI</defaultValue>
```

Comment:
Can't set this as the default value, I'd leave it blank

Reply:
Did you address this?

Reply:
addressed